pkg/app/reconcile_timer.go

```diff
@@ -61,11 +72,14 @@ func (rt ReconcileTimer) IsReadyAt(timeAt time.Time) bool {
 }
 
 func (rt ReconcileTimer) syncPeriod() time.Duration {
-	const defaultSyncPeriod = 30 * time.Second
-	if sp := rt.app.Spec.SyncPeriod; sp != nil && sp.Duration > defaultSyncPeriod {
-		return sp.Duration
+	selectedDur := rt.opts.DefaultSyncPeriod
+	if sp := rt.app.Spec.SyncPeriod; sp != nil {
```

Comment:
hey i thought Dmitriy told me one time that we don't like one liners like this, stylistically speaking

Reply:
generally for errors, this one was already there and it seems to scope variable.